python/pyspark/shell.py

```diff
@@ -86,7 +88,7 @@
     % (platform.python_version(), platform.python_build()[0], platform.python_build()[1])
 )
 if is_remote():
-    print("Client connected to the Spark Connect server at %s" % (os.environ["SPARK_REMOTE"]))
+    print("Client connected to the Spark Connect server at %s" % urlparse(os.environ["SPARK_REMOTE"]).netloc)
```

Comment:
I think this is fine; I don't know if security info would be in the URL (shouldn't be), and I wonder if some deployments want that info to clarify what's being connected to (like some kind of proxy), but can't think of a definite use case